src/azure-cli-core/azure/cli/core/auth/util.py

```diff
@@ -43,11 +43,7 @@ def aad_error_handler(error, **kwargs):
         recommendation = PASSWORD_CERTIFICATE_WARNING
     else:
         login_command = _generate_login_command(**kwargs)
-        recommendation = (
-            # Cloud Shell uses IMDS-like interface for implicit login. If getting token/cert failed,
-            # we let the user explicitly log in to AAD with MSAL.
-            "Please explicitly log in with:\n{}" if error.get('error') == 'broker_error'
-            else "Interactive authentication is needed. Please run:\n{}").format(login_command)
+        recommendation = f"Interactive authentication is needed. Please run:\n{login_command}"
```

Comment:
We may need to say something like "Run this command and add additional parameters as needed." because we can't predict how cx runs the command.